src/vuex/getters.js

```diff
@@ -110,7 +110,7 @@ export const depositDenom = getters =>
     : `uatom`
 
 // connection
-export const approvalRequired = state => state.connection.approvalRequired
+export const connection = state => state.connection
```

Comment:
Thinking about this. I believe we might be making too much use of getters, where we could just use `mapState` in the component.

Reply:
That's a great idea. Hadn't considered this.